ledger-tool/src/main.rs

```diff
@@ -82,6 +86,29 @@ fn main() {
         .subcommand(SubCommand::with_name("print").about("Print the ledger"))
         .subcommand(SubCommand::with_name("json").about("Print the ledger in JSON format"))
         .subcommand(SubCommand::with_name("verify").about("Verify the ledger's PoH"))
+        .subcommand(SubCommand::with_name("prune").about("Prune the ledger at the block height").arg(
+            Arg::with_name("valid_slot_list")
+                .long("valid-slot-list")
+                .value_name("FILENAME")
+                .takes_value(true)
+                .help("The location of the YAML file with a list of valid rollback slot heights and hashes"),
+        ))
+        .subcommand(SubCommand::with_name("list-roots").about("Output upto last <num-roots> root hashes and their heights starting at the given block height").arg(
```

Comment:
How about:
1) Add a `--slot-list <FILE>` arg, to write the roots out to a file that can be easily provide to `prune`
2) Rename `list-roots` command to `find-roots`?